client/web/src/enterprise/code-monitoring/CreateCodeMonitorPage.tsx

```diff
@@ -8,11 +8,29 @@ import { AuthenticatedUser } from '../../auth'
 import { BreadcrumbSetters, BreadcrumbsProps } from '../../components/Breadcrumbs'
 import { PageHeader } from '../../components/PageHeader'
 import { PageTitle } from '../../components/PageTitle'
+import classnames from 'classnames'
 
 export interface CreateCodeMonitorPageProps extends BreadcrumbsProps, BreadcrumbSetters {
     location: H.Location
-    authenticatedUser: AuthenticatedUser | null
+    authenticatedUser: AuthenticatedUser
 }
+
+interface FormCompletionSteps {
+    triggerCompleted: boolean
+    actionCompleted: boolean
+}
+
+interface Action {
+    recipient: string
+    enabled: boolean
+}
+interface CodeMonitorFields {
+    description: string
```

Comment:
```suggestion
    title: string
```
Can we call it `title` to match the field name? Or is this the name in the GraphQL schema?

Reply:
That's right, this is the name in the schema